shared/src/androidMain/kotlin/dev/johnoreilly/confetti/work/SessionNotificationSender.kt

```diff
@@ -31,25 +34,32 @@ class SessionNotificationSender(
 ) {
 
     suspend fun sendNotification() {
-        // If there is no user signed-in, no reason to process all sessions.
+        // If there is no signed-in user, skip.
         val user = authentication.currentUser.value ?: return
 
         val conferenceId = repository.getConference()
 
-        val sessions = repository.sessions(
+        val sessionsResponse = repository.sessions(
             conference = conferenceId,
             uid = user.uid,
             tokenProvider = user,
             fetchPolicy = FetchPolicy.CacheFirst,
         )
+
+        val sessions = sessionsResponse
             .data
             ?.sessions
             ?.nodes
-            ?.map { it.sessionDetails }
+            ?.map { query -> query.sessionDetails }
             .orEmpty()
 
-        // If current date is not in the conference range, no reason to process all sessions.
-        if (sessions.any { session -> session.startsAt.date == dateService.now().date }) {
+        // If there are no available sessions, skip.
+        if (sessions.isEmpty()) {
+            return
+        }
+
+        // If current date is not in the conference range, skip.
+        if (sessions.none { session -> session.startsAt.date == dateService.now().date }) {
```

Comment:
We were wrongly using `any` instead of `none`, I think I added it last minute to reduce processing and didn't test well in my previous PR.